app/src/androidTest/java/org/simple/clinic/overdue/AppointmentRepositoryAndroidTest.kt

```diff
@@ -348,23 +372,126 @@ class AppointmentRepositoryAndroidTest {
   @Test
   fun when_removing_appointment_with_reason_as_patient_already_visited_then_appointment_should_be_marked_as_visited() {
     val patientId = UUID.randomUUID()
-    val appointmentDate = LocalDate.now()
-    repository.schedule(patientId, appointmentDate).blockingAwait()
+    val appointmentDate = LocalDate.now(clock)
+    appointmentRepository.schedule(patientId, appointmentDate).blockingAwait()
 
-    val appointments = repository.recordsWithSyncStatus(SyncStatus.PENDING).blockingGet()
+    val appointments = appointmentRepository.recordsWithSyncStatus(SyncStatus.PENDING).blockingGet()
     assertThat(appointments).hasSize(1)
     assertThat(appointments.first().status).isEqualTo(Appointment.Status.SCHEDULED)
 
     val uuid = appointments[0].uuid
-    repository.setSyncStatus(listOf(uuid), SyncStatus.DONE).blockingGet()
-    repository.markAsVisited(uuid).blockingGet()
+    appointmentRepository.setSyncStatus(listOf(uuid), SyncStatus.DONE).blockingGet()
+    appointmentRepository.markAsVisited(uuid).blockingGet()
 
-    val updatedList = repository.recordsWithSyncStatus(SyncStatus.PENDING).blockingGet()
+    val updatedList = appointmentRepository.recordsWithSyncStatus(SyncStatus.PENDING).blockingGet()
     assertThat(updatedList).hasSize(1)
     updatedList[0].apply {
       assertThat(this.uuid).isEqualTo(uuid)
       assertThat(this.cancelReason).isNull()
       assertThat(this.status).isEqualTo(Appointment.Status.VISITED)
     }
   }
+
+  @Test
+  @Suppress("ASSIGNING_SINGLE_ELEMENT_TO_VARARG_IN_NAMED_FORM_FUNCTION")
+  fun high_risk_patients_should_be_present_at_the_top() {
+    data class BP(val systolic: Int, val diastolic: Int)
+
+    fun savePatientAndAppointment(
+        fullName: String,
+        hasHadStroke: Boolean = false,
+        hasDiabetes: Boolean = false,
+        hasHadKidneyDisease: Boolean = false,
+        age: String = "30",
+        vararg bpMeasurements: BP
+    ) {
+      val patientUuid1 = patientRepository.saveOngoingEntry(testData.ongoingPatientEntry(fullName = fullName, age = age))
```

Comment:
This can be just called `patientUuid` right? I'm assuming this was something leftover from a refactoring.

Reply:
Whoops, copy pasta! Fixing this.

Reply:
Has it been fixed? I still see `patientUuid1`.